src/test/java/net/smartcosmos/dao/objects/impl/ObjectPersistenceServiceTest.java

```diff
@@ -108,8 +113,208 @@ public void getObjects() throws Exception {
     }
 
     @Test
-    public void findByQueryParameters() throws Exception {
-        // TODO to.. do...
+    public void findByQueryParametersStringParameters() throws Exception
+    {
+        findByQueryParametersUtil();
+
+        Map<String, Object> queryParams = new HashMap<>();
+
+        queryParams.put(ObjectPersistenceService.OBJECT_URN_LIKE, "objectUrnQueryParams");
```

Comment:
It seems this can be accomplished with less work and cover the boundary/edge conditions and the common results, see comment below.